examples/ethereum/config/config.go

```diff
@@ -247,7 +253,12 @@ func LoadConfiguration() (*configuration.Configuration, error) {
 	//if err != nil || len(portValue) == 0 || port <= 0 {
 	//	return nil, fmt.Errorf("%w: unable to parse port %s", err, portValue)
 	//}
-	config.Port = 8080
+
+	tokenFilter := os.Getenv(TokenFilterEnv)
+	tokenFilterValue, err := strconv.ParseBool(tokenFilter)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
should we stop the process since it's a Fatal error? If it's not Fatal, we might need to lower down the error category. To decide if this is fatal or not - will this application work without this value being a bool?